package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "start": "sequelize db:migrate && node ./bin/www",
-    "test": "jest --passWithNoTests",
+    "test": "jest --passWithNoTests --forceExit",
```

Comment:
Does it make sense to have separate commands for unit tests and all tests?  In case integration tests take awhile and we want a tighter loop when developing?  I am open to what you both think, just a suggestion.

Reply:
I never had enough tests to make the command take too long (in a previous project ~50 tests for routes runs in a few seconds) but it might be a good idea!

Reply:
I've worked in projects where some End-to-End tests took some minutes to run, and it was very annoying. In our case, the whole thing took less than a minute, so I believe it's fine to have them all in a single command for now. But whenever we feel integration tests or any other kind of test is taking too long, then we should definitely separate them.

Reply:
Sounds good to me.  In a previous role, integration tests took almost an hour, so they were run rarely :(